modules/ROOT/pages/api-led-test.adoc

```diff
@@ -4,23 +4,23 @@ include::_attributes.adoc[]
 endif::[]
 
 So far, you've created an API and tested the HTTP request with a third-party client, and then published the API to Exchange.
-Now you'll use Studio to add validation and a unit test. In Studio, you'll use the scaffolding already created to:
+Now you'll use Studio to add validation and a unit test. In Studio, you'll use the scaffolding that's already created to:
 
 * Validate input using the Validation module.
 * Create an error handler for one error condition.
 
 == Step 4.1: Validation
 
 Add an operation from the Validation module to the API implementation to ensure that user input is appropriate for the API.
-Your API is very simple, so your testing will be simple. You'll test whether the size of the payload is within an acceptable range.
+Your API is very simple, so your testing is also simple. You'll test whether the size of the payload is within an acceptable range.
 
-Add the *Validate size* operation to the `hello-world` project:
+Add the *Validate size* operation to the `hello-world-app` project:
 
-. In the `hello-world` project in Studio, open `hello-world.xml` if it isn't already open, and make sure you are viewing the *Message Flow* tab of the canvas.
-. In *Mule Palette*, select *Add Modules > Validation* and drag it to the Studio canvas.
+. In the `hello-world-app` project in Studio, open `hello-world-api.xml` if it isn't already open, and make sure you are viewing the *Message Flow* tab of the canvas.
+. In *Mule Palette*, click *Add Modules > Validation* and drag it to the *Mule Palette*.
 +
 image:validation-module.png[View of Mule Palette with the Validation module selected]
-. From the list of operations displayed, drag *Validate size* to the Studio canvas and drop it to the right of *Set Payload*, inside the endpoint module.
+. From the list of operations, drag *Validate size* to the Studio canvas and drop it inside the endpoint module to the right of *Set Payload*.
```

Comment:
Is Vale OK with "right"?

Reply:
It flagged it, but in this case, there is no avoiding the use of *right*. For the flow to work, the logger has to be on the right side of the Set Payload component. Nothing prevents you from putting it on the left side, but if you did, that would be a different flow.